test/test-values-full.yaml

```diff
@@ -0,0 +1,37 @@
+---
+_000_commonConfig:
+  serialNumber: "default"  # usually the hostname
+  datasources:
+    sensorconnect:
+      enabled: true
+      iprange: "10.43.13.37/24"  # IP Range in CIDR notation
+      enableKafka: false
```

Comment:
why is kafka disabled?

Reply:
Because I want messages to be sent via MQTT, in order to use the MQTT connector in NodeRed instead of installing the Kafka plugin

Reply:
@JeremyTheocharis is this ok? or do I enable them both?

Reply:
Please do it with Kafka instead. The messages should still appear in MQTT via the bridge, so you can still use the MQTT plugin